libs/utils/utils.js

```diff
@@ -871,8 +871,11 @@ export async function loadArea(area = document) {
     window.dispatchEvent(new Event('milo:LCP:loaded'));
 
     // Post LCP operations.
-    if (isDoc && section.el.dataset.idx === '0') { loadPostLCP(config); }
-
+    if (window.PostLCPevnt && isDoc && section.el.dataset.idx === '0') {
+      window.addEventListener(config.PostLCPevnt, () => {
+        loadPostLCP(config);
+      });
+    } else if (isDoc && section.el.dataset.idx === '0') { loadPostLCP(config); }
```

Comment:
```suggestion
    if (isDoc && section.el.dataset.idx === '0') {
      if (window.postLCPevent) {
        window.addEventListener(window.postLCPevent, () => {
          loadPostLCP(config);
        });
      } else {
        loadPostLCP(config);
      }
    }
```

Reply:
well, it's mostly whitespace changes. If you prefer, remove the newlines or even the curly braces around lines 877 and 880. Don't know what your style is, but it should minify to roughly the same thing.
Not repeating the if condition seems like better DRY.